hack/datadog-gpu-monitoring.py

```
@@ -0,0 +1,193 @@
+WAS_ENABLED = ANNOTATION_PREFIX + "datadog-gpu-was-enabled"
+HOLDERS = ANNOTATION_PREFIX + "datadog-gpu-holders"
+
+# (kubectl resource, path to the gpu feature block)
+KINDS = (
+    ("datadogagentprofiles.datadoghq.com", ("spec", "config", "features", "gpu")),
+    ("datadogagents.datadoghq.com", ("spec", "features", "gpu")),
+)
+
+
+def kubectl(*args: str, check: bool = True) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(["kubectl", *args], text=True, capture_output=True, check=check)
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)